lib/_debug_agent.js

```diff
@@ -21,7 +21,7 @@ exports.start = function start() {
   agent.listen(process._debugAPI.port, function() {
     var addr = this.address();
     process._rawDebug('Debugger listening on port %d', addr.port);
-    process._debugAPI.notifyListen();
+    process._debugAPI.notifyListen(addr.port);
```

Comment:
`net.Socket#listen()` doesn't work that way, it emits an 'error' event when it can't bind to a port.